lua/lazyvim/plugins/extras/util/gh.lua

```diff
@@ -0,0 +1,64 @@
+return {
+
+  -- depends on the git extra for highlighting and auto-completion of github issues/prs
+  { import = "lazyvim.plugins.extras.lang.git" },
+
+  -- gh.nvim
+  {
+    "ldelossa/gh.nvim",
+    dependencies = {
+      {
+        "ldelossa/litee.nvim",
+        config = function()
+          require("litee.lib").setup()
+        end,
+      },
+    },
+    event = "VeryLazy",
```

Comment:
Is this needed? You're already defining `keys`, which means that the plugin will be lazy-loaded on the keymaps that the user will press. I'm genuinely asking because I haven't used the plugin itself. Is it needed to load on `VeryLazy` instead of when the user presses one of the mappings?

Reply:
I tried the PR locally and it also works without `event = "VeryLazy"` from my minimal testing. I don't think there's a reason to have this when the plugin can also be loaded on `keys`.

Also thank you for bringing this plugin to my attention. Wasn't aware of this and will give it a go :smile: